client/js/test/table-view.test.js

```diff
@@ -13,6 +13,35 @@ describe('table-view', () => {
 		document.documentElement.innerHTML = html;
 	});
 
+	describe('table footer', () => {
+		it('it has right size', () => {
+			//set up the inital state
+			const numCols = 1;
+			const numRows = 10;
+			const model = new TableModel(numCols, numRows);
+			const view = new TableView(model);
+			view.init();
+
+			//inspect the inital state
+			let ths = document.querySelectorAll('TFOOT TR');
+			expect(ths.length).toBe(numCols);
+
+		});
+
+		it('has to add up', () => {
```

Comment:
Don't know what "has to add up" means. I'm another dev who hasn't read your code and is working with your tests. How do I debug from here if the test fails?